### PR TITLE
test: enhance consumer integration tests

### PR DESCRIPTION
**Pull request by dherges**
## I'm submitting a...

```
[ ] Bug Fix
[ ] Feature
[x] Other (Refactoring, Added tests, Documentation, ...)
```

## Checklist

- [x] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
- [x] Tests for the changes have been added

## Description

Enhance coverage of integration tests.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: integration/consumers/tsc/src/main.ts [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+import { CustomConsulerModule } from './custom-consumer/custom-consumer.module';
+//import { JsxConsumerModule } from './jsx-consumer/jsx-consumer.module';
+import { MaterialConsumerModule } from './material-consumer/material-consumer.module';
+import { ModuleImportDirective } from './module-imports';
 
 @NgModule({
+  imports: [
+    CoreConsumerModule,
+    CustomConsulerModule,
+    //JsxConsumerModule,
```
Comment (dherges) [inline integration/consumers/tsc/src/main.ts]: @KyleMcNutt When commented-in, it produces a build error:

```bash
$ ngc -p integration/consumers/tsc/tsconfig.json
: 'react-integration-test' is not a known element:
1. If 'react-integration-test' is an Angular component, then verify that it is part of this module.
2. If 'react-integration-test' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas'of this component to suppress this message. ("
[ERROR ->]<react-integration-test></react-integration-test>
```

Can you maybe take a look?

File: integration/consumers/tsc/src/jsx-consumer/jsx-consumer.module.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ReactIntegrationModule } from '@sample/jsx';
+import { JsxConsumerComponent } from './jsx-consumer.component';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    ReactIntegrationModule
```
Comment (dherges) [inline integration/consumers/tsc/src/jsx-consumer/jsx-consumer.module.ts]: What's wrong here?

File: integration/consumers/tsc/src/jsx-consumer/jsx-consumer.component.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'jsx-consumer',
+  template: `
+<react-integration-test></react-integration-test>
```
Comment (dherges) [inline integration/consumers/tsc/src/jsx-consumer/jsx-consumer.component.ts]: What's wrong here?

**Comment by KyleMcNutt** [comment]
I'll try and take a look soon, it looks like whatever module React Integration Test is in isn't exporting it though?

**Comment by KyleMcNutt** [comment]
@dherges See the new PR I just opened for this:

https://github.com/dherges/ng-packagr/pull/439

**Pull request merged**